thirdeye/thirdeye-frontend/app/styles/components/rootcause-anomaly.scss

```diff
@@ -1,4 +1,12 @@
 .rootcause-anomaly {
+  &-row {
+    vertical-align: middle;
+  }
```

Comment:
This seems like a modifier, which means the name should be prefixed with `--` according to BEM [documentation](https://en.bem.info/methodology/naming-convention/#block-modifier-name). I would suggest a better name for `-row` would be `--vertical-align`.

Reply:
fixed